internal/grpc_service_config/config.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package grpc_service_config
```

Comment:
Should probably just be config or grpcconfig.

https://blog.golang.org/package-names (no underscores in Go package names).

Reply:
Yeah, this isn't a great package name. But can I do that in a follow up PR? It would change the existing directories and files and all of the existing imports for the raw protobuf message and it would kind of distract from the code changes here.